dataactbroker/permissions.py

```diff
@@ -31,21 +24,11 @@ def decorated_function(*args, **kwargs):
             try:
                 sess = GlobalDB.db().session
                 error_message = "Login Required"
-                if permission == "check_email_token":
-                    if LoginSession.isRegistering(session):
-                        return f(*args, **kwargs)
-                    else :
-                        error_message = "unauthorized"
-                elif permission == "check_password_token":
-                    if LoginSession.isResetingPassword(session):
-                        return f(*args, **kwargs)
-                    else :
-                        error_message  = "unauthorized"
-                elif LoginSession.isLogin(session):
+                if LoginSession.isLogin(session):
```

Comment:
Where is "session" defined and do we still need to pass it around? How is this session different than the sess you defined above it? More of a clarification question for me.